pkg/bindings/test/images_test.go

```diff
@@ -89,4 +89,39 @@ var _ = Describe("Podman images", func() {
 
 	})
 
+	//Tests to validate the image tag command.
```

Comment:
hey @sujil02 this is a super pedantic nit, and can be addressed later, but typically I prefer to have a space between the `//` and the comment, and the rest of libpod conforms to this. Could you make an update PR to fix these instances?

Reply:
Hi,

Thank you for letting me know, I am working on a PR to add more test cases in the same file. And I will take care of this along with that PR.